src/Lumi/Components/Upload.purs

```diff
@@ -539,17 +544,41 @@ upload = make component { initialState, render }
                             }
                         , children: []
                         }
+                    _ ->
+                      case variant of
+                        Avatar ->
+                          lumiUploadAvatarImage
+                            { style: R.css {}
+                            , children: [ userSvg ]
+                            }
+                        Logo ->
+                          lumiUploadAvatarImage
+                            { style: R.css { backgroundColor: cssStringHSLA colors.black4 }
+                            , children: []
+                            }
+                        _ ->
+                          lumiUploadAvatarImage
+                            { style: R.css {}
+                            , children: [ userSvg ]
+                            }
+
+                _, _ ->
+                  case variant of
+                    Avatar ->
```

Comment:
The `Avatar` case is superfluous, as it's subsumed by `_` down below and has the same result.